### ISSUE TITLE
Blocks only shown if there is an empty line at the end of the config file

### ISSUE THREAD

**Issue by tobalr** [regolith-linux/regolith-i3xrocks-config#46]
Hi
Thank you for your great work! I do enjoy using regolith instead of installing i3 on my own.

I have had some issues regarding custom i3xrocks configs. I have tried some of the examples on the https://github.com/regolith-linux/i3xrocks eg. this one:

```
# Print seconds since 1970-01-01
[epoch]
command=date +%s
interval=1         
```
The block would never show up.
After some testing I have found that if I add an empty line at the end of the file it works fine
```
# Print seconds since 1970-01-01
[epoch]
command=date +%s
interval=1
         
```

I expect that this is not intended behaviour  as the configs in this repo does not have an empty line at the end?

Version name: i3xrocks 1.3.2 © 2020 Ken Gilmer

Best regards, Tobias

**Comment by kgilmer** [comment]
Hi @tobalr , thanks and happy you are liking Regolith!  Yes, you're right that each conf.d file needs to have a empty line.  This is because i3xrocks merges all of the files together when it starts up so I think without the linefeed multiple lines will be put into one.  I took a look at the conf.d files in git, and they all have a linefeed at the end.  Ex: 

```bash
$ cat /etc/regolith/i3xrocks/conf.d/90_time 
# Date Time
[time]
interval=10
$
```

Maybe I am misunderstanding you?

**Comment by kgilmer** [comment]
Actually `disk-capacity` did not so I will fix it now.

**Comment by tobalr** [comment]
Hi @kgilmer . That makes sense. 
Sorry I missed the LF on the files in this repo. It looks like github does not show the last empty line when the last char is a LF.
I initially followed the examples from the README.adoc https://github.com/regolith-linux/i3xrocks#interval where the the individual example snippets does not all show a LF at the end. When testing the examples on the page some of them worked and so I started writing some custom blocks without knowing that it was required to have a LF. As you might guess I had some difficulties until I found out that it was the LF that made it work. 

It might just be me, but I think it would be easier to get going (not make the same mistake as I did) if there was a visible newline in the end of each individual example code block or a mention that this is required due to the merging somewhere in the doc.
